### ISSUE TITLE
Training on validation in minimal examples

### ISSUE THREAD

**Issue by eloygeenjaar** [catalyst-team/catalyst#909]
## 📚 Documentation

The 'minimal examples' train on the validation set instead of the training set.

**Comment by Scitator** [comment]
Looks like merged :)
